src/main/java/org/jsoup/safety/Allowlist.java

```diff
@@ -50,50 +50,50 @@ If you need to allow more through (please be careful!), tweak a base whitelist w
  </ul>
  
  <p>
- The cleaner and these whitelists assume that you want to clean a <code>body</code> fragment of HTML (to add user
+ The cleaner and these allowlists assume that you want to clean a <code>body</code> fragment of HTML (to add user
  supplied HTML into a templated page), and not to clean a full HTML document. If the latter is the case, either wrap the
- document HTML around the cleaned body HTML, or create a whitelist that allows <code>html</code> and <code>head</code>
+ document HTML around the cleaned body HTML, or create an allowlist that allows <code>html</code> and <code>head</code>
  elements as appropriate.
  </p>
  <p>
- If you are going to extend a whitelist, please be very careful. Make sure you understand what attributes may lead to
+ If you are going to extend an allowlist, please be very careful. Make sure you understand what attributes may lead to
  XSS attack vectors. URL attributes are particularly vulnerable and require careful validation. See 
  http://ha.ckers.org/xss.html for some XSS attack examples.
  </p>
 
  @author Jonathan Hedley
  */
-public class Whitelist {
+public class Allowlist {
```

Comment:
this is a big BC Break, changes like this should be done gradually with deprecations and a migration path

Reply:
This is why I proposed to keep `Whitelist` as a deprecated alias for a couple of releases.

Reply:
I've added the @deprecated Whitelist class as suggested.

Reply:
IMHO this is not enough, you also have to have overloaded variants of methods where you've changed the signature